tests/foreman/cli/test_leapp_client.py

```diff
@@ -0,0 +1,263 @@
+"""Test for LEAPP cli
+
+:Requirement: leapp
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Leappintegration
+
+:Team: Rocket
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:CaseAutomation: Automated
+
+:Upstream: No
+"""
+import pytest
+from wait_for import wait_for
+from wrapanapi.entities.vm import VmState
+
+from robottelo.constants import PRDS
+
+CUSTOM_REPOSET = {
+    'rhel8_8_bos': 'Red Hat Enterprise Linux 8 for x86_64 - BaseOS (RPMs)',
```

Comment:
Please merge this dict into CUSTOM_REPOS dict so that you end up with:

```
CUSTOM_REPOS = {
  'rhel8_8_bos': {
     'id':
     'name':
    'reposet':
    'releasever':
  }
...
}
```

Reply:
- Nice suggestion Lukash, I will take this in upcoming PR.

Reply:
@vijaysawant It should be this PR if you want to receive ACKs

Reply:
- Not sure about **merging**, we have only one upgrade path implemented which is 8.8 -> 9.2
- I am planning to keep both dictionaries, `CUSTOM_REPOSET` and `CUSOTM_REPOS` separate and these 2 dictionaries will be updated by adding more data related to another upgrade path like 7.9 to 8.9, etc in future.
- Pros: easy to understand major & minor versions which we are using by it's keys
- Cons: Few syntactical changes would be required in for loop between lines 170 to 179 and requires additional conditional statement base upon **source rhel os version**
- Hence I am keeping these 2 dictionaries separate as of now, but Merging both dictionaries can consider once updated.
- Talking about **moving** into the constants, then we can move it, but need to keep them separately because here we are using a specific reponame & release version (`major.minor`), I don't see any other constants which are specific to minor OS version (apart from kickstart)